ale_python_interface/ale_c_wrapper.cpp

```diff
@@ -1,3 +1,20 @@
-//all code is currently in the .h file
 #include "ale_c_wrapper.h"
 
+#include <cstring>
+#include <string>
+
+void encodeState(ALEState *state, char *buf) {
+	std::string str = state->serialize();
+
+	memcpy(buf, str.data(), str.length());
```

Comment:
Good catch. I'd like to merge into master & tag asap, so we can also just make it an issue for now, & fix it after releasing 0.5.1. @mhauskn who may want to weigh in on this.